client/src/components/landing/RegisterButton.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <button
+      v-if="buttonNumber === 2"
+      class="btn btn-primary"
+      @click="register()">
+      {{ $t('LandingPage.Nav.EndingPageButton') }}
+    </button>
+    <button
+      v-else
+      class="btn btn-primary"
+      @click="register()">
+      {{ $t('LandingPage.Nav.GetStarted') }}
+    </button>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({
+  name: 'RegisterButton',
+  props: {
+    buttonNumber: {
+      required: false,
+      type: Number,
+    },
+  },
+  methods: {
+    register(): void {
+      const zmienna = 0;
+    },
```

Comment:
I will add this code in seperate task, Just added it to Jira. Could you leave `// TODO: Add registration path retrieval` instead of `xonst zmienna = 0;`?